backend/app/crud.py

```diff
@@ -17,15 +17,11 @@ def get_user_by_email(db: Session, user_email: str) -> models.User:
     return db.query(models.User).filter(models.User.user_email == user_email).first()
 
 
-def get_user_by_name(db: Session, username: str) -> models.User:
```

Comment:
Ok so since the username is completely deleted from the User model now as it should have been a little while back, we need to probably go off the email for finding the user for login. I updated this to reflect that change by getting rid of this function and using the get_user_by_email function instead.